tests/test_compile_error.rs

```diff
@@ -1,6 +1,6 @@
 #![cfg(feature = "macros")]
 
-#[rustversion::stable]
+#[rustversion::not(nightly)]
```

Comment:
I've recently started using `beta` for library dev to pick up upcoming issues earlier :)